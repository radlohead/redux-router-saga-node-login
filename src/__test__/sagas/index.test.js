import root from '../../sagas';

it('sagas test', () => {
    const saga = root();
    expect(saga.next()).toMatchSnapshot();
});