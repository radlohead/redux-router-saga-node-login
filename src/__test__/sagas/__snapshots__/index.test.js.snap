// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sagas test 1`] = `
Object {
  "done": false,
  "value": Object {
    "@@redux-saga/IO": true,
    "combinator": true,
    "payload": Array [
      Object {
        "@@redux-saga/IO": true,
        "combinator": false,
        "payload": Object {
          "args": Array [],
          "context": null,
          "fn": [Function],
        },
        "type": "FORK",
      },
    ],
    "type": "ALL",
  },
}
`;
